ui-components/src/global/vendor/AgPagination.tsx

```diff
@@ -45,33 +47,15 @@ const AgPagination = ({
     dataPagerClassName.push("sticky")
```

Comment:
I don't think the `flex flex-col md:flex-row` classes on line 43 are actually doing anything, so maybe we should remove them?

Reply:
Done!